### PR TITLE
add libpq/{11.9, 10.14, 9.6.19}

### PR DESCRIPTION
**Pull request by n-bes**
- add libpq/11.9
- add libpq/10.14
- add libpq/9.6.19
- enable ```zlib``` and ```openssl``` by default
-------
- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libpq/all/conanfile.py [hunk 1]
```diff
@@ -18,7 +18,13 @@ class LibpqConan(ConanFile):
         "with_zlib": [True, False],
         "with_openssl": [True, False],
         "disable_rpath": [True, False]}
-    default_options = {'shared': False, 'fPIC': True, 'with_zlib': True, 'with_openssl': False, 'disable_rpath': False}
+    default_options = {
+        'shared': False,
+        'fPIC': False,
+        'with_zlib': True,
+        'with_openssl': True,
```
Comment (prince-chrismc) [inline recipes/libpq/all/conanfile.py:25]: I dont think we are allowed to change the defaults 🙊 I'll let the CCI team comment

**Review by prince-chrismc — COMMENTED** [review]
Typically we discourage older versions from being added (the CCI reports they are almost never used)

Do you explicitly need them?

File: recipes/libpq/all/conanfile.py (hunk 1 above)
Comment (n-bes) [reply recipes/libpq/all/conanfile.py:25]: Other recipes allowed it:

https://github.com/conan-io/conan-center-index/blob/master/recipes/pcre/all/conanfile.py#L39
https://github.com/conan-io/conan-center-index/blob/master/recipes/librhash/all/conanfile.py#L24
https://github.com/conan-io/conan-center-index/blob/master/recipes/boost/all/conanfile.py#L79

**Comment by n-bes** [comment]
> Typically we discourage older versions from being added (the CCI reports they are almost never used)
> 
> Do you explicitly need them?

No. Main idea that they are [supported](https://www.postgresql.org/support/versioning/):

image

File: recipes/libpq/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/libpq/all/conanfile.py:25]: In the diff on GitHub it looks like only `openssl` went from False to True. 

The value itself is fine, my concern is that it was changed.

Let's wait to see if the CCI team has a strong opinion.

**Comment by SpaceIm** [comment]
At least we can see that `with_openssl=True` is broken in this recipe. Seems that system libs of openssl are not properly injected in `build()`, leading to undefined references.

File: recipes/libpq/all/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/libpq/all/conanfile.py:25]: If it's the default value of upstream build system, it's fine.

**Comment by SpaceIm** [comment]
Adding those lines in build(), if visual studio, might help:

```python
            system_libs = ", ".join(["'{}.lib'".format(lib) for lib in self.deps_cpp_info.system_libs])
            tools.replace_in_file(os.path.join(self._source_subfolder, "src", "tools", "msvc", "Project.pm"),
                                  "libraries             => [],",
                                  "libraries             => [{}],".format(system_libs))
```

**Comment by SpaceIm** [comment]
Could you try https://github.com/n-bes/conan-center-index/pull/3?

File: recipes/libpq/all/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/libpq/all/conanfile.py:25]: Actually, I checked default value in 13.0, and all options are disabled by default except zlib, so default value of openssl option should be disabled in this recipe.

**Comment by n-bes** [comment]
> Could you try [n-bes#3](https://github.com/n-bes/conan-center-index/pull/3)?

All my test passed 👍

**Review by prince-chrismc — DISMISSED** [review]

**Review by SSE4 — DISMISSED** [review]

File: recipes/libpq/config.yml [hunk 2]
```diff
@@ -9,5 +9,11 @@ versions:
     folder: all
   "11.7":
     folder: all
+  "11.9":
+    folder: all
```
Comment (SpaceIm) [inline recipes/libpq/config.yml:13]:
```suggestion
  "11.9":
    folder: all
  "11.7":
    folder: all
```

**Comment by SpaceIm** [comment]
Stress test for CI ? :smile:

**Comment by n-bes** [comment]
Oops.... i just rebase / merge with master...

**Comment by prince-chrismc** [comment]
> Oops.... i just rebase / merge with master... What happened with CI?

You broke it! 😆 the CCI team will have to check into the problem on their end, we will have to wait

**Comment by SpaceIm** [comment]
It's not a CI issue, your merge is broken.

**Comment by Croydon** [comment]
PR commits get squashed, but all commit messages get listed in the commit description. If this gets merged it will easily generate the largest commit message in CCI 😄

**Comment by n-bes** [comment]
I will recreate PR if it needs.

**Comment by prince-chrismc** [comment]
85% sure

```sh
$ git reset --mixed 0e39e2c0bbe8b07ed57710f372e173924ce420ea
$ git commit -a -m "adding all lts versions of libpq"
$ echo 'make sure everything is okay'
$ git push -f
```

**Comment by madebr** [comment]
Can you merge with or rebase on current master?

**Comment by n-bes** [comment]
Hope, i fixed... 🤞

**Review by prince-chrismc — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**